okapi-common/src/main/java/org/folio/okapi/common/HttpClientCached.java

```diff
@@ -0,0 +1,286 @@
+package org.folio.okapi.common;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.http.HttpMethod;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.TreeSet;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * HTTP client with cache .. Can be used in most case as a replacement of
+ * Vert.x {@link io.vertx.core.http.HttpClient}
+ *
+ * @author adam
```

Comment:
The list of authors that contributes code is tracked in the git commit history.
Do you want to duplicate that information in the source code? How do we ensure that it is kept up to date? Is "adam" unambiguous now and in future?

Reply:
Yes.. I'll remove it and the 8 in total Okapi. 67 in RMB by the way

Reply:
Thanks, resolved. (I cannot resolve this `@author` conversion because I don't have write permissions for this repository.)